.circleci/generate_circleci_config.py

```diff
@@ -12,10 +12,10 @@
 # This should match what is in tests/__init__.py
 # https://hub.docker.com/r/kindest/node/tags
 kube_versions = [
-    "1.21.14",
```

Comment:
Let's actually leave this in for now. EKS still has 1.21. We can remove it in March.